ADMIN.md

```diff
@@ -34,16 +31,15 @@ there will be a dedicated Spend Profile Manager service that will allow users to
 setup different profiles that map to different billing accounts, and to grant
 access to users.
 
-Until that new service is build, Workspace Manager recognizes a single spend
-profile per deployment. This single spend profile corresponds to a SAM resource.
-The name of this resource is specified in the Workspace Manager deployment
-configuration (e.g. Helm charts) but needs to be setup and managed manually in
-SAM (i.e. there are no WSM endpoints for managing spend profiles, you have to
-talk to SAM directly). The CLI provides convenience commands for this setup and
-management.
+Until that new service is build, Workspace Manager must hardcode spend profiles
```

Comment:
Change 34-39:

Until that new service is built, setting up a new spend profile entails:

- Adding WSM configuration (helm charts) which associates spend profile name with a billing account ID
- Creating the SAM spend profile resource with `terra spend create-profile`